src/ui.cxx

```diff
@@ -992,6 +992,8 @@ void GetParamFile(Options &opt)
                         opt.mpinprocswritesize = atoi(vbuff);
                     else if (strcmp(tbuff, "MPI_use_zcurve_mesh_decomposition")==0)
                         opt.impiusemesh = (atoi(vbuff)>0);
+		    else if (strcmp(tbuff, "MPI_use_tree_decomposition")==0)
+		        opt.impiusetree = (atoi(vbuff)>0);
```

Comment:
Had a comment about adding this config option. Does also require an update to allvars.cxx to ensure the meta data is written for the config option.